ansible/roles/stack-sunbird/defaults/main.yml

```diff
@@ -122,3 +122,9 @@ sunbird_portal_player_cdn_enabled:
 sunbird_portal_preview_cdn_url:
 cdn_file_path:
 sunbird_portal_cdn_blob_url:
+#learner-service
+sunbird_subdomain_keycloak_base_url:
+#cert-service
+DOMAIN_URL:
+ENC_SERVICE_URL:
```

Comment:
@manzarul Please remove all these changes and the below empty value in ansible/inventory/env/group_vars/all.yml
sunbird_subdomain_keycloak_base_url:
This value will be get set in private repo